### ISSUE TITLE
3D background

### ISSUE THREAD

**Issue by MikelSalazar** [opengeospatial/GeoPoseWeb#7]
While not as impressive as the WebXR version I want to create with the Sandbox, it might be interesting to create a background in 3D to add more interactivity in those computer that can hadle it. For reference, here is an example I created: https://systembd.com

**Comment by janerivi** [comment]
I think if used in a discrete way as part of some graphics at a footer that might something that illustrates geopose without interfeering with legibility of text etc it can be be fine. I liked the way that systembd has 3d effect near the footer. If it is toned down enough and don't disturbe reading of content it can be cool. I advice against having large movement animation running continuously behind text/navigation items. Witht the effect at the footer it stops animating if you stop scrolling - that seems like the level that something could work.

**Comment by MikelSalazar** [comment]
Correct, since it would be a background, it must be something simple as to not interfere with the text. Imagine a 3D globe slowly rotating on a corner of the screen or a starry sky moving as you scroll down to give the illusion of depth.
